library/net.c

```diff
@@ -92,6 +92,14 @@ static int wsa_init_done = 0;
 
 #endif /* ( _WIN32 || _WIN32_WCE ) && !EFIX64 && !EFI32 */
 
+/* Some MS functions want int and MSVC warns if we pass size_t,
+ * but the standard fucntions use socklen_t, so cast only for MSVC */
+#if defined(_MSC_VER)
```

Comment:
Yes, I just saw that. Agreed, we can fix the typo in 2.1 and development on other occasions.